pages/apim/3.x/kubernetes/apim-kubernetes-controller-overview.adoc

```diff
@@ -0,0 +1,23 @@
+[[apim-kubernetes-controller-overview]]
+= Kubernetes Controller plugin introduction
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_kubernetes_controller_overview.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+
+[label label-version]#New in version 3.7#
+
+== Overview
+
+APIM 3.7 introduced a Kubernetes Controller plugin for APIM Gateway allowing the deployment of APIs using https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Custom Resource Definitions (CRDs)^].
```

Comment:
@mouligno This is not a deprecated things to use this controller ? It seems to me that it is no longer maintained since 2 years or am I wrong?

Reply:
you're right and it's not working with APIM 3.9+

Reply:
so could we add a commit to remove all of this in documentation 😁 🙏 it will be already done when I arrive with my big hoof to remove the code too 🙈

Reply:
I'll schedule this in a separate Jira ticket and we can address after the release once confirmed.

Reply:
https://graviteedevops.atlassian.net/browse/DOC-232